src/main/java/javapoz24/team3/rental/domain/rental/Rental.java

```diff
@@ -0,0 +1,28 @@
+package javapoz24.team3.rental.domain.rental;
+
+import javapoz24.team3.rental.domain.base.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import java.net.URI;
+import java.util.Set;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Rental extends BaseEntity {
+
+    private String name;
+    private URI webDomain;
```

Comment:
Wydaje mi się że tutaj powinno być raczej pole typu String a nie URI bo nie każda baza danych to łyknie, ale nie jestem tego pewien w 100%.

Reply:
Na razie zrobię więc String, potem się najwyżej zobaczy.